.gitignore

```diff
@@ -689,3 +689,10 @@ hs_err_pid*
 !/gradle/wrapper/gradle-wrapper.jar
 
 # End of https://www.toptal.com/developers/gitignore/api/visualstudiocode,intellij,reactnative,xcode,node,macos,android,androidstudio,ruby,cocoapods,buck
```

Comment:
I'd regenerated the whole file with the tool used originally and update this list in the comment as well.

Reply:
Agreed.

Reply:
Cool, I didn't know this tool, thanks for the suggestion, fixed.